### DISCUSSION TITLE
Support for Azure Key Vault + CSI secret store driver in KEDA

### DISCUSSION THREAD

**Discussion by ezYakaEagle442** [kedacore/keda-docs#379] [Q&A] [answered]
KEDA secret management with Azure Key Vault + CSI secret store driver is not documented at [https://keda.sh/docs/2.1/concepts/authentication/#secrets](https://keda.sh/docs/2.1/concepts/authentication/#secrets)

**Comment by jeffhollan** [comment, marked as answer]
Not supported today, but I'd love to see KEDA TriggerAuth extended to know how to deal with this CSI pattern that Key Vault users https://github.com/kedacore/keda/issues/900
